packages/@aws-cdk/aws-ecs/lib/container-definition.ts

```diff
@@ -469,15 +482,15 @@ export class ContainerDefinition extends cdk.Construct {
       image: this.imageConfig.imageName,
       memory: this.props.memoryLimitMiB,
       memoryReservation: this.props.memoryReservationMiB,
```

Comment:
Shouldn’t these be Lazy? What happens if you add something to the array after initialization?